src/System.Private.CoreLib/shared/Internal/Win32/RegistryKey.cs

```diff
@@ -214,18 +215,18 @@ public unsafe string[] GetValueNames()
             return names.ToArray();
         }
 
-        public object GetValue(string name)
+        public object? GetValue(string name)
         {
             return GetValue(name, null);
         }
 
-        public object GetValue(string name, object defaultValue)
+        public object? GetValue(string name, object? defaultValue)
```

Comment:
This should be annotated with a TODO-NULLABLE comment for https://github.com/dotnet/roslyn/issues/26761, as it can return null only if defaultValue is null.